source/index.html.md

```diff
@@ -630,6 +630,31 @@ object][ApplePay-PaymentToken] for more information.
 </p>
 
 
+##### Method: `samsungpay`
+
+Samsung Pay provides the payment details as either JWE or JWE/JWS JOSE objects. Currenlty we only
+accept the JWE.
+Your system must extract the content encryption key (<code>cek</code>) used to encrypt the JWE
+payload and Base64-encode it, along with the entire JWE.
+
+<dl class="dl-vertical">
+  <dt>samsungpay[jwe]</dt>
```

Comment:
```suggestion
  <dt>samsungpay[token]</dt>
```

... or something alike? Granted, it's JWE, but we could also use `samsungpay[json]` which I also dislike. What's the **name** of that fellah in Samsung Pay lingo?

Reply:
Would we like to receive "the JWE/JWS" in another parameter? What would the name of that one be?

Reply:
It's not JSON. I think `jwt` [JSON web token](https://en.wikipedia.org/wiki/JSON_Web_Token) could be the correct lingo.

Reply:
But then there's overloading, since JWT can both refer to JWS and JWE.